spec/units/transcoding_spec.rb

```diff
@@ -57,53 +57,53 @@ def generate_special_derivatives
 
   after(:all) do
     Object.send(:remove_const, :GenericFile);
-    Object.send(:remove_const, :ContentDatastream);
+    Object.send(:remove_const, :original_fileDatastream);
```

Comment:
It's removing a constant that was never defined in the first place.  You _do_ define ContentDatastream and should remove that constant.